### PR TITLE
replace build.jl by Artifacts.toml

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
I'm still a bit shaky if that really works out. So if someone could test this branch, who uses this feature, I would be happy.

Fix: https://github.com/JuliaPlots/Plots.jl/issues/1783

### CODE REVIEW

**Comment by daschw** [comment]
What exactly should be tested? If plotly works when building with this branch?

**Review by daschw — APPROVED** [review]
Works for me, also with `ENV["PLOTS_HOST_DEPENDENCY_LOCAL"] = "true"`

**Comment by BeastyBlacksmith** [comment]
I don't really know when this environment variable is actually used... I guess it might be an IJulia thing. Building shouldn't be necessary anymore.

**Comment by daschw** [comment]
Yes, works for me also in IJulia with `ENV["PLOTS_HOST_DEPENDENCY_LOCAL"] = true`.

**Comment by BeastyBlacksmith** [comment]
Great, thanks for checking!

**Comment by KristofferC** [comment]
nice!

**Comment by fredrikekre** [comment]
I am pretty sure this does not work; artifact downloads has to be .tar.gz?

File: src/init.jl [hunk 1]
```diff
@@ -74,16 +76,22 @@ function __init__()
         end
     end
 
-    if haskey(ENV, "PLOTS_HOST_DEPENDENCY_LOCAL")
-        use_local_plotlyjs[] = ENV["PLOTS_HOST_DEPENDENCY_LOCAL"] == "true"
-        use_local_dependencies[] = isfile(plotly_local_file_path) && use_local_plotlyjs[]
-        if use_local_plotlyjs[] && !isfile(plotly_local_file_path)
-            @warn("PLOTS_HOST_DEPENDENCY_LOCAL is set to true, but no local plotly file found. run Pkg.build(\"Plots\") and make sure PLOTS_HOST_DEPENDENCY_LOCAL is set to true")
+    if get(ENV, "PLOTS_HOST_DEPENDENCY_LOCAL", "false") == true
```
Comment (fredrikekre) [inline src/init.jl:79]: This path is never taken, since what you get back is a the string `"true"` if you have set it to that, so that is never `== true`

**Comment by BeastyBlacksmith** [comment]
> I am pretty sure this does not work; artifact downloads has to be .tar.gz?

From https://github.com/JuliaLang/Pkg.jl/issues/1921 and the iris exampe I got the impression it does work this way, if I don't use the artifact string macro

File: src/init.jl (hunk 1 above)
Comment (BeastyBlacksmith) [reply src/init.jl:79]: I changed that in https://github.com/JuliaPlots/Plots.jl/pull/3067

**Pull request merged**